### PR TITLE
[Sql Server] (No.1) Add Sql Server module and list instances

### PR DESCRIPTION
**Pull request by shenqianjin**
image

image

### CODE REVIEW

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        List<ISqlServer> servers = com.microsoft.azure.toolkit.lib.Azure.az(AzureSqlServer.class).sqlServers();
+        servers.parallelStream()
+            .filter(server -> Objects.nonNull(server.entity()))
+            .map(server -> new SqlServerNode(this, server.entity().getSubscriptionId(), server))
+            .forEach(this::addChildNode);
+    }
+
+    @Override
+    public void removeNode(String sid, String id, Node node) {
+        MySQLMvpModel.delete(sid, id);
```
Comment (andxu) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java:48]: mysql?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SubscriptionsDialog.java [hunk 2]
```diff
@@ -159,9 +156,19 @@ private void createUIComponents() {
         AnActionButton refreshAction = new AnActionButton("Refresh", AllIcons.Actions.Refresh) {
             @Override
             public void actionPerformed(AnActionEvent anActionEvent) {
+                this.setEnabled(false);
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SubscriptionsDialog.java:159]: Why there are code changes for authentication? May need update the feature branch?

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/AzureModule.java [hunk 3]
```diff
@@ -163,6 +167,9 @@ protected void refreshItems() throws AzureCmdException {
         if (!isDirectChild(mysqlModule)) {
             addChildNode(mysqlModule);
         }
+        if (!isDirectChild(sqlServerModule)) {
+            addChildNode(sqlServerModule);
+        }
```
Comment (Flanker32) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/AzureModule.java:172]: Please call module refresh in `refreshFromAzure()` below

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    @Override
+    public @Nullable AzureIconSymbol getIconSymbol() {
+        return AzureIconSymbol.SqlServer.MODULE;
+    }
+
+    @Override
+    protected void refreshItems() throws AzureCmdException {
+        List<ISqlServer> servers = com.microsoft.azure.toolkit.lib.Azure.az(AzureSqlServer.class).sqlServers();
```
Comment (Flanker32) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/sqlserver/SqlServerModule.java:39]: Seems no need to use full name here

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SubscriptionsDialog.java (hunk 2 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SubscriptionsDialog.java]: this is not changed by me. i have already aligned my  base branch with develop.

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**